test/acceptor_SUITE.erl

```diff
@@ -94,7 +94,7 @@ groups() ->
 
 misc_bad_transport(_) ->
 	doc("Reject invalid transport modules."),
-	{error, badarg} = ranch:start_listener(misc_bad_transport,
+	{error, {badtransport, bad_transport}} = ranch:start_listener(misc_bad_transport,
```

Comment:
If renaming `badtransport` then we probably want to rename the module to `invalid_transport` or something to make things less confusing.